src/Features/Core/Portable/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -222,7 +224,8 @@ private int NonParamsParameterCount(IMethodSymbol method)
             }
             else
             {
-                var name = semanticFacts.GenerateNameForExpression(semanticModel, expression);
+                var name = semanticFacts.GenerateNameForExpression(
+                    semanticModel, expression, capitalize: false, cancellationToken: cancellationToken);
```

Comment:
Sorry about that, in both cases I was asking why `capitalize` was hard-coded as false.